lib/resources/forms.js

```diff
@@ -69,6 +69,13 @@ module.exports = (service, endpoint) => {
     /* eslint-enable indent */
       // if we don't have managed encryption, or the form carries its own key,
       // we can use the form xml as-is. otherwise we must inject things.
+      .then((partial) => ((/\.(xlsx?|xml)$/.test(partial.xmlFormId))
```

Comment:
the only suggestion i'd make is to consider using `rejectIf` if you feel like it. it does exist and it's used a couple places but there's nothing wrong w this really.

Reply:
ok, i tried using rejectIf